premake/loong.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index 01eb384..3d17698 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -13,6 +13,29 @@ license=('BSD')
+ depends=('glibc')
+ source=("https://github.com/premake/premake-core/releases/download/v${_pkgver}/premake-${_pkgver}-src.zip")
+ sha512sums=('a349dabe3db5b503c283e836da89a5239007696f87756fe34e011bee655c86cd8ccc1c0c00f72a46d0d10302c2f5d67e37d566313e68a16406924a8ba70c5d8f')
++makedepends=('dos2unix')
++
++# add loongarch64 support
++source+=(
++  "premake-rescv.patch::https://github.com/premake/premake-core/commit/610924f549c1f61233fe1f5ab930b8bbc4489755.patch"
```

Comment:
Use merged one: https://github.com/premake/premake-core/commit/82c9d90495940e2d0d574e1c7849e9698f23b090

Or just rebase our fix to the version 5.0.0-beta3